api/src/db/DynamoBusinessesDataClient.ts

```diff
@@ -155,7 +145,7 @@ export const DynamoBusinessesDataClient = (
     return await Promise.all(
       Items.map(async (object: any): Promise<Business> => {
         const data = unmarshall(object).data;
```

Comment:
sand - we could return `unmarshall(object).data;`

Reply:
I see the suggestion, but we're not returning unmarshall(result.Items[0]) directly. Instead, we are returning the data inside the result of unmarshall(result.Items[0]). Since unmarshall is a method call, I prefer to assign it to a variable first for better clarity and readability. This makes the data transformation more explicit, and I think it improves the overall maintainability of the code. Additionally, this approach keeps up with the consistency of the other data clients.